frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsListCard.component.tsx

```diff
@@ -47,6 +47,7 @@ export const TicketsListCard = () => {
 			containerOrFederation,
 			isFederation,
 		);
+		TicketsActionsDispatchers.fetchRiskCategories(teamspace);
```

Comment:
If this is moved to `tickets.component.tsx` it can be invoked inside a `useEffect` without dependencies and it would send many less requests to the backend. You can do that because the riskCategories depend on the teamspace rather than a ticket or a template.

The same may be also true for fetchTemplates and fetchTickets, but of course, in a different useEffect as you need the `containerOrFederation` as a dependency

```ts
// in tickets.component.tsx

	useEffect(() => { TicketsActionsDispatchers.fetchRiskCategories(teamspace); }, []);
	useEffect(() => {
		TicketsActionsDispatchers.fetchTickets(
			teamspace,
			project,
			containerOrFederation,
			isFederation,
		);
		TicketsActionsDispatchers.fetchTemplates(
			teamspace,
			project,
			containerOrFederation,
			isFederation,
		);
		TicketsActionsDispatchers.fetchRiskCategories(teamspace);
	}, [containerOrFederation]);
```